config_utilities/CMakeLists.txt

```diff
@@ -84,6 +90,10 @@ install(
   LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(DIRECTORY introspection_viewer/ DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(PROGRAMS introspection_viewer/introspection_viewer.py
+        RENAME config_utilities_viewer
```

Comment:
minor preference for `config-utilities-viewer` to match `composite-configs` stylistically